arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -1029,7 +1014,13 @@
             "in": "query",
             "required": false,
             "type": "string",
-            "description": "The expand expression to apply on the operation."
+            "description": "The expand expression to apply on the operation.",
+			"enum": [
+              "instanceView"
+            ],
+			"x-ms-enum": {
+              "name": "InstanceViewTypes"
```

Comment:
This PR https://github.com/Azure/autorest/pull/828 for single value enum was merged. 
The [documentation](https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#single-value-enum-as-a-constant) should also help clarify things.